features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/application-authentication.xml.j2

```diff
@@ -151,4 +151,16 @@
         <AuthenticationEndpointRedirectParam name="{{parameter}}"/>
         {% endfor %}
     </AuthenticationEndpointRedirectParams>
+    {% if tenant.domainDropDown.enable is defined %}
+      <TenantDomainDropDownEnabled>{{tenant.domainDropDown.enable}}</TenantDomainDropDownEnabled>
+    {% endif %}
+    {% if oauth.custom_grant_type is defined %}
```

Comment:
This should be corrected as below:
 {% if tenant.data_listener_urls is defined %}
      <TenantDataListenerURLs>
      {% for listener in tenant.data_listener_urls %}   
           <TenantDataListenerURL>
           {{listener.url}}
           </TenantDataListenerURL>     
       {% endfor %}
        </TenantDataListenerURLs>
     {% endif %}